### ISSUE TITLE
256color support is missing

### ISSUE THREAD

**Issue by ruturajv** [arcticicestudio/nord-dircolors#4]
If I use the suggested dir_colors along with `nord-gnome-terminal` I miss out on all coloring !

OS: Fedora release 23 (Twenty Three)
Terminal: GNOME Terminal 3.18.3

**Comment by arcticicestudio** [comment]
Sorry to hear that, but I was not able to reproduce this :/

Do you installed Nord GNOME Terminal by following the [install](https://github.com/arcticicestudio/nord-gnome-terminal#installation) and [activation](https://github.com/arcticicestudio/nord-gnome-terminal#activation) sections?
The install script should create a new profile named `Nord` which must be used to enable the color palette.
![](https://raw.githubusercontent.com/arcticicestudio/nord-gnome-terminal/develop/src/assets/scrot-readme-lazy-profile-change.png)

Do you installed Nord dircolors manually or via the [install.sh](https://github.com/arcticicestudio/nord-dircolors/blob/develop/install.sh) script? Please make sure that the [dir_colors](https://github.com/arcticicestudio/nord-dircolors/blob/develop/src/dir_colors) file is placed at `~/.dir_colors` (note the **dot** infront of the file name).

If it still does not work please take a screenshot or screencast of your terminal showing the missing colorization and post the exact version of your GNOME Terminal (`gnome-terminal --version`).

**Comment by ruturajv** [comment]
I'm so sorry, its happening for `Fedora release 23 (Twenty Three)`. So again...

- If I install nord-gnome-terminal, and switch profile to Nord; everything is fine ! 👍 

- Along with the above configuration, If I also use the nord-dircolors, then I lose the coloring of different files that was working with `nord-gnome-terminal` and all files are shown as the white color. If I remove the `~/.dir_colors`, things are back to normal.  OR if I add `TERM xterm-256color` to `~/.dir_colors`, its OK.

- Yes, I installed using install.sh, as well as manually tried copying `dir_colors` as `~/.dir_colors`
- My gnome version is `GNOME Terminal 3.18.3`.

**Comment by ruturajv** [comment]
I'm updating my original comment to include Fedora instead of erroneously mentioned Ubuntu.

**Comment by arcticicestudio** [comment]
Thanks for the details 👍 
I'm running Arch Linux so I'm always using the latest package versions, that's the reason I always need version information for reproduction 😄 
I tried to reproduce your steps using
  - GNOME Terminal `3.22.1` with
    - ZSH `5.3.1` and
    - Bash `4.4.012` with
  - Linux `4.10.1` and coreutils `8.26` on
    - GNOME DE and
    - i3 WM

and everything worked fine.
Are there absolutely no colors, only files like code sources (`.js` or `.java` etc.) are white or even directories and/or symlinks are uncolored?

I'll create a VirtualBox for Fedora this weekend and let you know if I'm able to reproduce this, maybe this is a distibution related problem again 😒

**Comment by ruturajv** [comment]
varied files, like images, tarballs, which had color lost it ! 
By adding line `TERM xterm-256color` fixed the issue (atleast I thought, since the Nord color scheme returned to file colors)

**Comment by arcticicestudio** [comment]
I've create a VirtualBox with the same versions you provided and it worked fine.
Unfortunately I'm currently not able to help you with this problem since I can't reproduce it to find the origin of this problem 😞 

Are you sure there are no other scripts which maybe modify the dircolors too?
Is the `LS_COLORS` environment variable set and contains the definitions provided by the theme?
The output should be similiar to this:
```shell
echo $LS_COLORS
no=00:rs=0:fi=00:di=01;34:ln=36:mh=04;36:pi=04;01;36:so=04;33:do=04;01;36:bd=01;33:cd=33:or=31:mi=01;37;41:ex=01;36:su=01;04;37:sg=01;04;37:ca=01;37:tw=01;37;44:ow=01;04;34:st=04;37;44:*.7z=01;32:*.ace=01;32:*.alz=01;32:*.arc=01;32:*.arj=01;32:*.bz=01;32:*.bz2=01;32:*.cab=01;32:*.cpio=01;32:*.deb=01;32:*.dz=01;32:*.ear=01;32:*.gz=01;32:*.jar=01;32:*.lha=01;32:*.lrz=01;32:*.lz=01;32:*.lz4=01;32:*.lzh=01;32:*.lzma=01;32:*.lzo=01;32:*.rar=01;32:*.rpm=01;32:*.rz=01;32:*.sar=01;32:*.t7z=01;32:*.tar=01;32:*.taz=01;32:*.tbz=01;32:*.tbz2=01;32:*.tgz=01;32:*.tlz=01;32:*.txz=01;32:*.tz=01;32:*.tzo=01;32:*.tzst=01;32:*.war=01;32:*.xz=01;32:*.z=01;32:*.Z=01;32:*.zip=01;32:*.zoo=01;32:*.zst=01;32:*.aac=32:*.au=32:*.flac=32:*.m4a=32:*.mid=32:*.midi=32:*.mka=32:*.mp3=32:*.mpa=32:*.mpeg=32:*.mpg=32:*.ogg=32:*.opus=32:*.ra=32:*.wav=32:*.3des=01;35:*.aes=01;35:*.gpg=01;35:*.pgp=01;35:*.doc=32:*.docx=32:*.dot=32:*.odg=32:*.odp=32:*.ods=32:*.odt=32:*.otg=32:*.otp=32:*.ots=32:*.ott=32:*.pdf=32:*.ppt=32:*.pptx=32:*.xls=32:*.xlsx=32:*.app=01;36:*.bat=01;36:*.btm=01;36:*.cmd=01;36:*.com=01;36:*.exe=01;36:*.reg=01;36:*~=02;37:*.bak=02;37:*.BAK=02;37:*.log=02;37:*.log=02;37:*.old=02;37:*.OLD=02;37:*.orig=02;37:*.ORIG=02;37:*.swo=02;37:*.swp=02;37:*.bmp=32:*.cgm=32:*.dl=32:*.dvi=32:*.emf=32:*.eps=32:*.gif=32:*.jpeg=32:*.jpg=32:*.JPG=32:*.mng=32:*.pbm=32:*.pcx=32:*.pgm=32:*.png=32:*.PNG=32:*.ppm=32:*.pps=32:*.ppsx=32:*.ps=32:*.svg=32:*.svgz=32:*.tga=32:*.tif=32:*.tiff=32:*.xbm=32:*.xcf=32:*.xpm=32:*.xwd=32:*.xwd=32:*.yuv=32:*.anx=32:*.asf=32:*.avi=32:*.axv=32:*.flc=32:*.fli=32:*.flv=32:*.gl=32:*.m2v=32:*.m4v=32:*.mkv=32:*.mov=32:*.MOV=32:*.mp4=32:*.mpeg=32:*.mpg=32:*.nuv=32:*.ogm=32:*.ogv=32:*.ogx=32:*.qt=32:*.rm=32:*.rmvb=32:*.swf=32:*.vob=32:*.webm=32:*.wmv=32:
```

If *not* there must be another script which also changes the `LS_COLORS`env var. Maybe you've put it into your dotfiles (`~/.bashrc`, `~/.zshrc` etc.) and forgot to remove it after installing the theme?

**Comment by ruturajv** [comment]
Strange, here is some more debugging o/p

```
$ echo $LS_COLORS #no dircolors
rs=0:di=38;5;33:ln=38;5;51:mh=00:pi=40;38;5;11:so=38;5;13:do=38;5;5:bd=48;5;232;38;5;11:cd=48;5;232;38;5;3:or=48;5;232;38;5;9:mi=01;05;37;41:su=48;5;196;38;5;15:sg=48;5;11;38;5;16:ca=48;5;196;38;5;226:tw=48;5;10;38;5;16:ow=48;5;10;38;5;21:st=48;5;21;38;5;15:ex=38;5;40:*.tar=38;5;9:*.tgz=38;5;9:*.arc=38;5;9:*.arj=38;5;9:*.taz=38;5;9:*.lha=38;5;9:*.lz4=38;5;9:*.lzh=38;5;9:*.lzma=38;5;9:*.tlz=38;5;9:*.txz=38;5;9:*.tzo=38;5;9:*.t7z=38;5;9:*.zip=38;5;9:*.z=38;5;9:*.Z=38;5;9:*.dz=38;5;9:*.gz=38;5;9:*.lrz=38;5;9:*.lz=38;5;9:*.lzo=38;5;9:*.xz=38;5;9:*.bz2=38;5;9:*.bz=38;5;9:*.tbz=38;5;9:*.tbz2=38;5;9:*.tz=38;5;9:*.deb=38;5;9:*.rpm=38;5;9:*.jar=38;5;9:*.war=38;5;9:*.ear=38;5;9:*.sar=38;5;9:*.rar=38;5;9:*.alz=38;5;9:*.ace=38;5;9:*.zoo=38;5;9:*.cpio=38;5;9:*.7z=38;5;9:*.rz=38;5;9:*.cab=38;5;9:*.jpg=38;5;13:*.jpeg=38;5;13:*.gif=38;5;13:*.bmp=38;5;13:*.pbm=38;5;13:*.pgm=38;5;13:*.ppm=38;5;13:*.tga=38;5;13:*.xbm=38;5;13:*.xpm=38;5;13:*.tif=38;5;13:*.tiff=38;5;13:*.png=38;5;13:*.svg=38;5;13:*.svgz=38;5;13:*.mng=38;5;13:*.pcx=38;5;13:*.mov=38;5;13:*.mpg=38;5;13:*.mpeg=38;5;13:*.m2v=38;5;13:*.mkv=38;5;13:*.webm=38;5;13:*.ogm=38;5;13:*.mp4=38;5;13:*.m4v=38;5;13:*.mp4v=38;5;13:*.vob=38;5;13:*.qt=38;5;13:*.nuv=38;5;13:*.wmv=38;5;13:*.asf=38;5;13:*.rm=38;5;13:*.rmvb=38;5;13:*.flc=38;5;13:*.avi=38;5;13:*.fli=38;5;13:*.flv=38;5;13:*.gl=38;5;13:*.dl=38;5;13:*.xcf=38;5;13:*.xwd=38;5;13:*.yuv=38;5;13:*.cgm=38;5;13:*.emf=38;5;13:*.ogv=38;5;13:*.ogx=38;5;13:*.aac=38;5;45:*.au=38;5;45:*.flac=38;5;45:*.m4a=38;5;45:*.mid=38;5;45:*.midi=38;5;45:*.mka=38;5;45:*.mp3=38;5;45:*.mpc=38;5;45:*.ogg=38;5;45:*.ra=38;5;45:*.wav=38;5;45:*.oga=38;5;45:*.opus=38;5;45:*.spx=38;5;45:*.xspf=38;5;45:
```

```
$ echo $LS_COLORS #with dircolors
rs=0:di=38;5;33:ln=38;5;51:mh=00:pi=40;38;5;11:so=38;5;13:do=38;5;5:bd=48;5;232;38;5;11:cd=48;5;232;38;5;3:or=48;5;232;38;5;9:mi=01;05;37;41:su=48;5;196;38;5;15:sg=48;5;11;38;5;16:ca=48;5;196;38;5;226:tw=48;5;10;38;5;16:ow=48;5;10;38;5;21:st=48;5;21;38;5;15:ex=38;5;40:*.tar=38;5;9:*.tgz=38;5;9:*.arc=38;5;9:*.arj=38;5;9:*.taz=38;5;9:*.lha=38;5;9:*.lz4=38;5;9:*.lzh=38;5;9:*.lzma=38;5;9:*.tlz=38;5;9:*.txz=38;5;9:*.tzo=38;5;9:*.t7z=38;5;9:*.zip=38;5;9:*.z=38;5;9:*.Z=38;5;9:*.dz=38;5;9:*.gz=38;5;9:*.lrz=38;5;9:*.lz=38;5;9:*.lzo=38;5;9:*.xz=38;5;9:*.bz2=38;5;9:*.bz=38;5;9:*.tbz=38;5;9:*.tbz2=38;5;9:*.tz=38;5;9:*.deb=38;5;9:*.rpm=38;5;9:*.jar=38;5;9:*.war=38;5;9:*.ear=38;5;9:*.sar=38;5;9:*.rar=38;5;9:*.alz=38;5;9:*.ace=38;5;9:*.zoo=38;5;9:*.cpio=38;5;9:*.7z=38;5;9:*.rz=38;5;9:*.cab=38;5;9:*.jpg=38;5;13:*.jpeg=38;5;13:*.gif=38;5;13:*.bmp=38;5;13:*.pbm=38;5;13:*.pgm=38;5;13:*.ppm=38;5;13:*.tga=38;5;13:*.xbm=38;5;13:*.xpm=38;5;13:*.tif=38;5;13:*.tiff=38;5;13:*.png=38;5;13:*.svg=38;5;13:*.svgz=38;5;13:*.mng=38;5;13:*.pcx=38;5;13:*.mov=38;5;13:*.mpg=38;5;13:*.mpeg=38;5;13:*.m2v=38;5;13:*.mkv=38;5;13:*.webm=38;5;13:*.ogm=38;5;13:*.mp4=38;5;13:*.m4v=38;5;13:*.mp4v=38;5;13:*.vob=38;5;13:*.qt=38;5;13:*.nuv=38;5;13:*.wmv=38;5;13:*.asf=38;5;13:*.rm=38;5;13:*.rmvb=38;5;13:*.flc=38;5;13:*.avi=38;5;13:*.fli=38;5;13:*.flv=38;5;13:*.gl=38;5;13:*.dl=38;5;13:*.xcf=38;5;13:*.xwd=38;5;13:*.yuv=38;5;13:*.cgm=38;5;13:*.emf=38;5;13:*.ogv=38;5;13:*.ogx=38;5;13:*.aac=38;5;45:*.au=38;5;45:*.flac=38;5;45:*.m4a=38;5;45:*.mid=38;5;45:*.midi=38;5;45:*.mka=38;5;45:*.mp3=38;5;45:*.mpc=38;5;45:*.ogg=38;5;45:*.ra=38;5;45:*.wav=38;5;45:*.oga=38;5;45:*.opus=38;5;45:*.spx=38;5;45:*.xspf=38;5;45:
```

![dir_colors](https://cloud.githubusercontent.com/assets/227817/24204162/ad037860-0f3d-11e7-9218-a0d3859b7f0a.png)

**Comment by arcticicestudio** [comment]
Sorry for the late commit.
I've added the missing `*-256color` `TERM` definitions which should fix all 256 color terminal emulators.
@ruturajv Can you please check out branch 🌱 [bugfix/ghi-#4-256color-support-is-missing](https://github.com/arcticicestudio/nord-dircolors/tree/bugfix/ghi-%234-256color-support-is-missing) to see if this fixes your problem?

**Comment by ruturajv** [comment]
Sure, I'll check it tom. on my office box.

**Comment by ruturajv** [comment]
Yes, This does seem to fix the issue :+1:

**Comment by arcticicestudio** [comment]
Thanks for the feedback 😉 
I've created PR #6 which will be merged and 🚢 shipped with the next release version.

**Comment by arcticicestudio** [comment]
@ruturajv Sorry for my late reply, I've been busy with my job.

I've merged #6 and :ship: shipped it with the new release version [0.1.1](https://github.com/arcticicestudio/nord-dircolors/releases/tag/v0.1.1).
